packages/pangea-node-sdk/src/services/authz.ts

```diff
@@ -0,0 +1,172 @@
+import PangeaResponse from "@src/response.js";
+import BaseService from "./base.js";
+import PangeaConfig from "@src/config.js";
+import { AuthZ } from "@src/types.js";
+
+/**
+ * AuthZService class provides methods for interacting with the AuthZ Service
+ * @extends BaseService
+ */
+class AuthZService extends BaseService {
+  constructor(token: string, config: PangeaConfig) {
+    super("authz", token, config);
+  }
+
+  /**
+   * @summary Tuple Create
```

Comment:
There is been a lot of debate on exact wording to be used to describe things, so these summaries and descriptions will most likely have to be updated. The api summary & descriptions also need updates, so there probably isn't a need to play too much with the text used in the SDK until the API descriptions are nailed down, and I don't think these text changes need to hold back the SDK updating, I'm just calling it out